File: acme-java-reactive/src/main/java/acme/AcmeBankApplication.java
package acme;

import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import io.swagger.v3.oas.annotations.info.Info;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@OpenAPIDefinition(
    info = @Info(
        title = "Acme Bank Reactive API",
        version = "1.0",
        description = "Acme Bank Reactive API documentation"
    )
)
@SpringBootApplication
public class AcmeBankApplication {

    public static void main(String[] args) {
        SpringApplication.run(AcmeBankApplication.class, args);
    }
}

File: acme-springboot-nonblocking/src/main/kotlin/acme/AcmeBankApplication.kt
package acme

import io.swagger.v3.oas.annotations.OpenAPIDefinition
import io.swagger.v3.oas.annotations.info.Info
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@OpenAPIDefinition(
    info = Info(
        title = "Acme Bank API (Non-Blocking)",
        version = "1.0",
        description = "Acme Bank API documentation - Non-blocking version with R2DBC"
    )
)
@SpringBootApplication
class AcmeBankApplication

fun main(args: Array<String>) {
    runApplication<AcmeBankApplication>(*args)
}
